tidy3d/web/api/tidy3d_stub.py

```diff
@@ -19,29 +20,31 @@
 from ..core.types import TaskType
 from ...components.simulation import Simulation
 from ...plugins.mode.mode_solver import ModeSolver
+from ...components.heat.simulation import HeatSimulation
+from ...components.heat.data.sim_data import HeatSimulationData
 
-
-SimulationType = Union[Simulation]
-SimulationDataType = Union[SimulationData]
+SimulationType = Union[Simulation, HeatSimulation]
+SimulationDataType = Union[SimulationData, HeatSimulationData]
 
 
 class Tidy3dStub(BaseModel, TaskStub):
 
-    simulation: SimulationType
+    simulation: SimulationType = pd.Field(discriminator="type")
```

Comment:
eventually we should add some more description here, if only to help other developers understand